src/parser/estree_translator.ml

```diff
@@ -510,6 +510,7 @@ end with type t = Impl.t) = struct
             "filter", option expression filter;
           ]
       | _loc, Identifier id -> identifier id
+      | loc, Literal ({ Literal.value = Ast.Literal.BigInt _; _ } as lit) -> bigint_literal (loc, lit)
```

Comment:
I didn't add new BigIntLiteral constructor, because it started to error across the project about missing cases, not only in parser

Reply:
To be clear, you don't need to add a `BigIntLiteral` constructor to the OCaml AST that the Flow parser generates. I'm talking about the JSON estree AST that `estree_translator` outputs. This is the JSON that shows up in the `.tree.json` test files. Looking at your PR, it looks like you've correctly updated this. For example, in `valid-binary.tree.json` I see

```
      "expression":{
        "type":"BigIntLiteral",
        "loc":{"source":null,"start":{"line":1,"column":0},"end":{"line":1,"column":12}},
        "range":[0,12],
        "value":null,
        "bigint":"0b101011101n"
      },
```

which conforms to the estree spec. So nice job!